graylog2-web-interface/src/views/components/widgets/SaveOrCancelButtons.jsx

```diff
@@ -8,11 +10,22 @@ type Props = {
   onFinish: () => void,
 };
 
-const SaveOrCancelButtons = ({ onFinish, onCancel }: Props) => (
-  <>
-    <Button onClick={onFinish} bsStyle="primary">Save</Button>
-    <Button onClick={onCancel}>Cancel</Button>
-  </>
-);
+const SaveOrCancelButtons = ({ onFinish, onCancel }: Props) => {
+  const { handleSubmit, dirty } = useFormikContext();
+  const _onFinish = useCallback((...args) => {
+    if (handleSubmit && dirty) {
+      handleSubmit();
+    }
+
+    return onFinish(...args);
+  }, [onFinish, handleSubmit]);
```

Comment:
The current implementation does not work as expected (the behaviour on save did not changed), looks the reason is that `dirty` is missing in this array.

Reply:
:white_check_mark: